### PR TITLE
Face detection api

### PR DESCRIPTION
**Pull request by usqr**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Shape Detection API demo
+
+Demo for the [article about Text Detection API](https://blog.comandeer.pl//toc-to-najprawdziwsza-twarz.html).
```
Comment (vokiel) [inline docs/README.md:3]: Double `//`

```suggestion
Demo for the [article about Text Detection API](https://blog.comandeer.pl/toc-to-najprawdziwsza-twarz.html).
```

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+		function markDetection(element) { // Function for marking the detected areas.
+			const lineWidth = element.landmarks ? 5 : 2;
+			canvasContext.strokeStyle = "lime";
+			canvasContext.lineWidth = lineWidth;
+			canvasContext.strokeRect(element.boundingBox.left, element.boundingBox.top, element.boundingBox.width, element.boundingBox.height);
+		}
+	</script>
+</body>
+
+</html>
```
Comment (vokiel) [inline docs/index.html:189]: Missing newline at the end of the file.

```suggestion
</html>
```

**Review by vokiel — COMMENTED** [review]
PR looks good: two small issues and missing favicon.

Can not really test it as this API seems to not be working on my Chrome@Linux

File: docs/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "private": true,
+  "version": "0.0.0",
+  "description": "Demo for the article about Text Detection API.",
+  "main": "detector.mjs",
+  "scripts": {
+    "start": "http-server ./"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Comandeer/text-detection-api-demo.git"
```
Comment (Comandeer) [inline docs/package.json:12]: All URL-s should be updated to point the new repo.

File: docs/index.html [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+	</div>
+	<script type="module">
+		import detectShape from './detector.mjs';
+		const el = document.querySelector('#imageDisplay');
+		const button = document.querySelector('input[type="file"]');
+		const canvas = document.querySelector("canvas");
+		const canvasContext = canvas.getContext('2d');
+		const infoText = document.querySelector(".info-text")
+
+		button.addEventListener('click', (event) => { event.target.value = null }) // Clearing the image - needed if you want to use the same one for different shape detections.
```
Comment (Comandeer) [inline docs/index.html:153]: Isn't it automatically done by the `change` listener?

**Review by Comandeer — CHANGES_REQUESTED** [review]
Overall looks good.

However we probably shouldn't put it directly in the `docs/` directory, but in its subdirectory, e.g. `docs/shape-detection-api-demo`. This way it will be easier to add demos for other articles in the future.

File: docs/index.html (hunk 4 above)
Comment (usqr) [reply docs/index.html:153]: Nope, If the file is the same it doesn't trigger `change`

File: docs/index.html [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+			<p>
+				Please bear in mind, this mechanism only works in Chrome and it requires this flag:
+				<code>chrome://flags#enable-experimental-web-platform-features</code>
+				to be turned on.
+			</p>
+			<p>
+
+			</p>
+		</div>
+		<canvas style="position: relative;" />
```
[the remark this answers is not in the document]
Comment (usqr) [reply docs/index.html]: fixed

File: docs/package.json (hunk 3 above)
Comment (usqr) [reply docs/package.json]: Should I link to the labs repo? :raised\_eyebrow:

**Review by Comandeer — APPROVED** [review]
LGTM!

**Pull request merged**